client/app/components/projects/planning-project-card.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@glimmer/component';
+
+const PACKAGE_STATUS_CODES = {
```

Comment:
Can you export these, too:

`export const PACKAGE_STATUS_CODES = {`

then import from somewhere else:

`import { PACKAGE_STATUS_CODES } from '../projects/planning-project-card';`